### ISSUE TITLE
Add HOWTO: Custom gradients in Flax

### ISSUE THREAD

**Issue by marcvanzee** [google/flax#897]
The PixelCNN has an example of using custom gradients: [pixelcnn.py](https://github.com/google/flax/blob/master/examples/pixelcnn/pixelcnn.py#L422).

[The Autodiff Cookbook](https://jax.readthedocs.io/en/latest/notebooks/autodiff_cookbook.html) by the JAX team is very useful for background reading as well.

**Comment by marcvanzee** [comment]
Closing this since there has been no interest in this HOWTO.
